### PR TITLE
add cypress names testing

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[speedup] Bio: Names modal (5)](https://trello.com/c/7mmo84WT/7216-speedup-bio-names-modal-5)

File: cypress/integration/my-orcid/my-orcid-top-bar.e2e.js [hunk 1]
```diff
@@ -31,39 +34,117 @@ describe('My Orcid top bar' + runInfo(), () => {
...
-      })
-      it('Drag and drop to rearrange', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('change privacy', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('make changes and cancel', () => {
-        // Expect changes NOT to be display outside and inside of the modal
+    before(() => {
```
Comment (leomendoza123) [inline cypress/integration/my-orcid/my-orcid-top-bar.e2e.js:37]: Please do not delete the test description that we are not implementing. 
This is meant to be developed later by someone else.

Comment (leomendoza123) [reply cypress/integration/my-orcid/my-orcid-top-bar.e2e.js:37]: like `change privacy` or `make changes and cancel`

File: cypress/support/clean.commands.js [hunk 2]
```diff
@@ -20,19 +47,53 @@ Cypress.Commands.add('cleanCountries', (url) => {
...
+        },
+      },
+      headers: {
+        'X-XSRF-TOKEN': cookie.value,
+      },
+    })
+  })
+})
+
+Cypress.Commands.add('cleanOtherNames', (url) => {
```
Comment (leomendoza123) [inline cypress/support/clean.commands.js:89]: We are not using the URL parameter.
(I am deleting the other ones on my last PR)

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**